test/BUILD.gn

```diff
@@ -59,18 +59,21 @@ static_library("brave_test_support_unit") {
   }
 }
 
-if (!is_ios) {
-group("other_unit_tests") {
+source_set("test_support") {
```

Comment:
`test_support` is added because the ipfs tests needs to depend on the same profile target and it would create a circular dependency otherwise.